effortless-chef-io/effortless-config.md

```diff
@@ -4,43 +4,51 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
 
 ## Patterns
 
-### Chef Repo Cookbook pattern
+### Chef Repo Cookbook Pattern
 
-This pattern is used to build policyfiles using the chef_repo pattern for organizing your cookbooks. The chef-repo pattern can be found [here](https://docs.chef.io/chef_repo/).
+This pattern builds Policyfiles using the chef-repo pattern for organizing your cookbooks. More information about the [chef-repo](https://docs.chef.io/chef_repo/) pattern can be found in our Chef Infra Documentation. View an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern).
```

Comment:
```suggestion
This pattern builds Policyfiles using the [chef-repo pattern](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern) for organizing your cookbooks. Our Chef Infra documentation contains more information about the [chef-repo](https://docs.chef.io/chef_repo/) pattern.
```

or something else that doesn't have a here link

Reply:
Okay that makes sense.

Reply:
```suggestion
This pattern uses the [chef-repo](https://docs.chef.io/chef_repo/) to store and organize everything you need to define your infrastructure with Chef Infra, including:

- Cookbooks (including recipes, attributes, custom resources, libraries, and templates)
- Data bags
- Policyfiles

The Effortless Config pattern builds Policyfiles using the chef-repo. View an [example chef-repo](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern) in the Chef Effortless GitHub repository.
```